keras/layers/merge.py

```diff
@@ -187,6 +187,25 @@ class Add(_Merge):
     It takes as input a list of tensors,
     all of the same shape, and returns
     a single tensor (also of the same shape).
+
+    # Examples
+
+    ```python
+        from keras.layers.merge import Add  # a layer
+        from keras.layers.merge import add  # a functional interface
+        from keras.layers import Input, Dense
+        from keras.models import Model
```

Comment:
For `Dense`, yes, but it seems necessary to make it clear that we can use
```python
import keras.layers.Add
from keras.layers.merge import Add
from keras.layers import Add
```
and if there's one that should be preferred, it should be mentioned somewhere (and perhaps why). I personally am still confused about it.